user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr :hoge
```

Comment:
__[rubocop]__
* Missing top-level class documentation comment.
* Do not use `attr`. Use `attr_reader` instead. [:ref](https://github.com/bbatsov/ruby-style-guide#attr)

<!-- Comment by [SideCI](https://sideci.com) -->